### PR TITLE
Adding multibrowser support

### PR DESCRIPTION
**Pull request by ouchadam**
turns out sass isn't injecting webkit/moz versions of our css, so I've manually added them back. Fixes the amazon tv issues we're having.

img_20161121_131706_1024

### CODE REVIEW

**Comment by florianmski** [comment]
Ugh? Really?

@frapontillo anything you could think of why sass isn't doing this?

**Comment by florianmski** [comment]
So apparently it doesn't work out of the box 😢 
We need to use `@mixin`, @ouchadam could you use something like this https://css-tricks.com/snippets/sass/mixin-prefix-properties/ ? Myabe hardcoding `webkit`, `moz`... would also be great 👍

**Comment by frapontillo** [comment]
Uh, my bad 😢

File: angular2-client/src/app/dashboards/xmas-countdown/xmas-countdown.component.scss [hunk 1]
```diff
@@ -31,10 +31,28 @@
...
+	-ms-animation: snow 10s linear infinite;
 	animation: snow 10s linear infinite;
 }
+
 @keyframes snow {
   0% {background-position: 0px 0px, 0px 0px, 0px 0px;}
   50% {background-position: 500px 500px, 100px 200px, -100px 150px;}
   100% {background-position: 500px 1000px, 200px 400px, -100px 300px;}
 }
+@-moz-keyframes snow {
```
Comment (xrigau) [inline angular2-client/src/app/dashboards/xmas-countdown/xmas-countdown.component.scss]: inb4 this is laggy

Comment (ouchadam) [inline angular2-client/src/app/dashboards/xmas-countdown/xmas-countdown.component.scss]: only on the fire stick, is fine with firefox on a pc 👅

File: angular2-client/src/app/dashboards/xmas-countdown/xmas-countdown.component.scss [hunk 2]
```diff
@@ -31,10 +31,28 @@
 	top: 0;
 	width: 100%;
 	z-index:1;
+  -webkit-animation: snow 10s linear infinite;
```
Comment (xrigau) [inline angular2-client/src/app/dashboards/xmas-countdown/xmas-countdown.component.scss]: there's an extra space 👀

**Comment by ouchadam** [comment]
@florianmski had a look into getting https://github.com/postcss/postcss working with `angular-cli-build.js` but it's over my head ¯\_(ツ)_/¯

**Comment by frapontillo** [comment]
@ouchadam nonnonononononoononnononoon if you want to do it automatically, use SCSS mixins, don't trust angular-cli on plugging **_ANYTHING_** into the build pipeline coz it doesn't work

**Comment by florianmski** [comment]
@ouchadam @frapontillo 
Made use of the mixins, your call 💃

File: angular2-client/src/app/dashboards/dashboards-common.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import "../mixins.scss";
```
Comment (xrigau) [inline angular2-client/src/app/dashboards/dashboards-common.scss]: 🌮

**Comment by xrigau** [comment]
👏  very cool stuff @florianmski @ouchadam :shipit:

**Pull request merged**